NEWS

```diff
@@ -1,13 +1,60 @@
-Release a.b
------------
+Release 1.12 (xx February 2021)
+-------------------------------
 
- * Samtools stats now displays the number of supplementary reads in the
+ * samtools stats now displays the number of supplementary reads in the
    SN section. Also, supplementary reads are no longer considered when
    splitting read pairs by orientation (inward, outward, other).
    
- * Samtools stats now counts only the filtered alignments that overlap
+ * samtools stats now counts only the filtered alignments that overlap
    target regions, if any are specified.
 
+ * samtools view now accepts option -N, which takes a file containing
+   read names of interest. This allows the output of only the reads with
+   names contained in the given file. Thanks to Daniel Cameron. (#1324)
+
+ * samtools view -d option now works without a tag associated value, which
+   allows it to output all the reads with the given tag. (#1339; fixed #1317)
+
+ * samtools view -d and -D options now accept integer values associated with
+   tags, not just strings. Thanks to `@dariome` for the suggestion. (#1357)
+
+ * samtools view now works with the filtering expressions introduced by HTSlib.
+   The filtering expression is passed to the program using option -e, e.g.
+   `samtools view -e '!(flag&2048)' align.bam` filters out all the supplementary
+   reads. (#1346)
```

Comment:
Possibly mention that other subcommands can do similar with `-Ofilter=expr` or so.

Reply:
This correct syntax would be e.g. `--input-fmt-option 'filter=!(flag&2048)'`.

Also I think this example isn't useful, unless we deliberately with to make the point it's identical to `-F 2048`.  I'd argue it's also better to write it as `!flag.supplementary` anyway as it's self documenting.

IMO a better example is something you cannot do with the existing filter options.  Eg looking for records with query-name ending in `#49` that have a mate aligned in a different chromosome. Ie:

```
samtools view -e 'qname =~ "#49$" && mrefid != refid'
```

Reply:
Oops, I knew I should have written out the long option (and then might have noticed the thinko). Thanks.